.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: required
 language: java
+dist: trusty # see: https://travis-ci.community/t/error-installing-oraclejdk8-expected-feature-release-number-in-range-of-9-to-14-but-got-8/3766/4
 jdk:
 - oraclejdk8
```

Comment:
I think we should better use `openjdk8` here since open source.

Reply:
8dbed86